src/pages/Gene.tsx

```diff
@@ -34,7 +34,7 @@ class Gene extends React.Component<IGeneProps>
     }
 
     private get loader() {
-        return loaderWithText("Fetching INSIGHT(ful) alterations...");
+        return loaderWithText("Fetching SIGNAL(ing) alterations...");
```

Comment:
SIGNAL(ing) does not sound as good as INSIGHT(ful)

Reply:
lol yeah prolly better to change to just `fetching alterations`, unless you have a better pun for signaldb